src/components/molecules/TheHeader.vue

```diff
@@ -41,16 +41,27 @@
         <!-- end lang toggle desktop -->
 
         <!-- start gc logo with mobile lang toggle -->
+        <!-- src="../../assets/sig-blk-en.svg" -->
         <!-- src={language === "en" ? "../../assets/sig-blk-en.svg" : "../../assets/sig-blk-fr.svg"} -->
         <div
           className="flex flex-col sm:flex-row container w-full mx-auto py-1 px-5 justify-between items-center"
         >
           <div className="flex flex-row items-center justify-between">
-            <img
-              className="h-5 xs:h-7 sm:h-8 md:h-9 mx-3 xs:mx-0"
-              src="../../assets/sig-blk-en.svg"
-              :alt="$t('canadaLogoHeader')"
-            />
+            <div v-if="changeLanguageTo === 'fr'">
```

Comment:
This is exactly what I was thinking :) the only thing is that I would remove both `<div>` and place the v-if and v-else on the `img` tag themselves.

Reply:
I will try that, I wasn't sure.